web/telemetry/registry.go

```diff
@@ -0,0 +1,10 @@
+package telemetry
+
+import "gorm.io/gorm"
+
+func NewTelemetryRegistry(db *gorm.DB) *TelemetryRegistry {
+	return &TelemetryRegistry{
+		"host_telemetry": NewHostTelemetryExtractor(db),
+		//"cluster": NewClusterTelemetryExtractor(),
```

Comment:
Leftover of showoff of this beautiful orthogonal design? :grimacing:

Reply:
I'd call it _an example_ :innocent:

Reply:
:100: